src/binary/ibc-setup/commands/init.spec.ts

```diff
@@ -0,0 +1,89 @@
+import fs from 'fs';
+
+import test from 'ava';
+import axios from 'axios';
+import sinon from 'sinon';
+
+import { Options, run } from './init';
+
+const consoleLog = sinon.stub(console, 'log');
```

Comment:
This will be replaced with a Winston logger, right?

Reply:
That's correct.